### DISCUSSION TITLE
「はいふんまいなす」と入力した際にハイフンマイナスが候補に出るようにしてほしい

### DISCUSSION THREAD

**Discussion by ghost** [google/mozc#773] [Ideas]
マイナス・ダッシュ・ハイフンマイナスなどと目視での判別が困難ないし不可能な記号が多数あるため、対応策として「はいふんまいなす」と入力した際に下記2つの記号が候補に出ると助かります。
- `-`U+002D、半角ハイフンマイナス
- `－` U+FF0D、全角ハイフンマイナス

**Comment by CRC32EX** [comment]
Why don't use dictionary tool???

![image](https://github.com/google/mozc/assets/25629617/612aac53-68e4-4671-8888-bad6af862bee)

![image](https://github.com/google/mozc/assets/25629617/22e5409f-8707-4678-b869-e022b6d41d9c)

**Comment by hiroyuki-komatsu** [comment]
Hi avalanchemarramrat,
Thank you for your request.

The candidate window shows annotations to distinguish the candidates.
Does it work for you?
![image](https://github.com/google/mozc/assets/11634377/d80d6ffb-841a-487a-a730-9517d4fedd74)


As CRC32EX mentioned, using a user dictionary is also another solution.
You can add annotations to words in the user dictionary.

Thank you,
